.env.example

```diff
@@ -0,0 +1,8 @@
+BLOB_ACCOUNT_NAME=
+BLOB_CONTAINER_NAME=
+BLOB_ACCOUNT_KEY=
+BLOB_SUB_ID=
+AML_COMPUTE_CLUSTER_NAME=
+AML_COMPUTE_CLUSTER_MIN_NODES=
```

Comment:
Might want to put the min_nodes as 0, so the AML compute cluster will not incur costs after idle period is reached

Reply:
added #366 to track since this is out of scope for the PR